studio/lib/interfaces/specialPages.ts

```diff
@@ -10,7 +8,7 @@ export interface CustomerCasePage {
   _updatedAt: string;
   basicTitle: string;
   page: string;
-  slug: Slug;
+  slug: string;
```

Comment:
Have this changed?

Reply:
Yep. Since the customer case page is field translated, the slug is returned as a string.